src/screens/TreeDetailsScreen.tsx

```diff
@@ -62,14 +74,23 @@ export default function TreeDetailsScreen({
     toggleEditing();
   };
 
+  const handleSaveComment = () => {
+    saveComment(comment);
+    addComment(comment, entry.uuid);
```

Comment:
if save comment adds a comment, what is the reason for another add comment function? could we not just have one text field/button that's just "add comment"